buildlib/rdma_functions.cmake

```diff
@@ -265,3 +265,13 @@ function(rdma_finalize_libs)
     endif()
   endforeach()
 endfunction()
+
+# Generate a pkg-config file
+function(rdma_pkg_config PC_LIB_NAME PC_LIB_PRIVATE PC_INCLUDE_PRIVATE PC_REQUIRES)
+  set(PC_LIB_NAME "${PC_LIB_NAME}")
+  set(PC_LIB_PRIVATE "${PC_LIB_PRIVATE}")
+  set(PC_INCLUDE_PRIVATE "${PC_INCLUDE_PRIVATE}")
+  set(PC_REQUIRES "${PC_REQUIRES}")
+  configure_file(${CMAKE_SOURCE_DIR}/buildlib/template.pc.in ${CMAKE_CURRENT_BINARY_DIR}/lib${PC_LIB_NAME}.pc @ONLY)
```

Comment:
Don't we need a build/lib/pkgconfig/  as well specifying options to 'use in place' ?

Reply:
rdma-core is designed to be built an run 'in-place' when the -DIN_PLACE=1 is used. It sets up rpaths and other things to enable it.

If you want to compile another application against it using pkgconfig I would expect you'd configure that application telling pkgconfig to search rdma-core/build/lib/pkgconfig for the .pc files before searching system directories, and then those .pc files should mirror the IN_PLACE setup and include -I and -L directives pointing to the build directory, not to system paths.

Reply:
Ok, will do.